haystack/pipelines/base.py

```diff
@@ -367,7 +368,8 @@ def eval(
         self,
         queries: List[str],
         labels: List[MultiLabel],
-        params: Optional[dict] = None
+        params: Optional[dict] = None,
+        sas_model_name_or_path: str = None
```

Comment:
Could we document a suggested model here please? Similar to what we have here: https://github.com/deepset-ai/haystack/blob/13510aa753d4e390f244398cc50654185dddbcde/haystack/nodes/evaluator/evaluator.py#L168

Most importantly:
```
- Large, powerful, but slow model for English only: "cross-encoder/stsb-roberta-large"
- Large model for German only: "deepset/gbert-large-sts"
- Good default for multiple languages: "sentence-transformers/paraphrase-multilingual-mpnet-base-v2"
```